supporting_content/dimensions/resourceid.md

```diff
@@ -0,0 +1,55 @@
+# Dimension: ResourceId
+
+## Example mappings
+
+| Provider  | Data set                 | Vendor column                        |
+|-----------|--------------------------|--------------------------------------|
+| AWS       | CUR                      | line_item_resource_id                |
+| GCP       | Big Query Billing Export | resource.name? resource.global_name? |
+| Microsoft | Cost details             | ResourceId                           |
+
+## Examples
```

Comment:
Do we need to qualify that these are existing examples and not suggested values when implementing FOCUS?

Reply:
Section name says Example Mappings which should be clear hopefully.